mail/mail.go

```diff
@@ -15,30 +15,31 @@ type Client struct {
 	Config config.Config
 }
 
-func (mc *Client) SendEmail(ctx context.Context, email, message string) error {
+func (mc *Client) SendEmail(ctx context.Context, email, subject, message string) error {
 	debug, ok := ctx.Value(settings.DebugKey).(bool)
 	if ok && debug {
-		return mc.SendEmailStub(email, message)
+		return mc.SendEmailStub(email, subject, message)
 	}
-	return mc.SendSMTPEmail(email, message)
+	return mc.SendSMTPEmail(email, subject, message)
 }
 
-func (mc *Client) SendEmailStub(email, message string) error {
+func (mc *Client) SendEmailStub(email, subject, message string) error {
 	slog.Info(
 		"sending email",
 		slog.Any("fromEmail", mc.Config.FromEmail),
 		slog.Any("toEmail", email),
+		slog.Any("subject", subject),
 		slog.Any("message", message),
 	)
 	return nil
 }
 
-func (mc *Client) SendSMTPEmail(email, message string) error {
+func (mc *Client) SendSMTPEmail(email, subject, message string) error {
```

Comment:
by the way, why don't you accept context as an argument here? Your mail client can also time out, so you need to give a caller an option to cancel request